questionnaires/evening_questionnaire/evening_questionnaire_armt.json

```diff
@@ -0,0 +1,704 @@
+[
+    {
+        "field_name": "medications",
+        "form_name": "evening_questionnaire",
+        "section_header": "Medication Compliance",
+        "field_type": "radio",
+        "field_label": "Did you take your medications in the last 24 hours ?",
+        "select_choices_or_calculations": [
+            {
+                "code": "0",
+                "label": "Took all doses "
+            },
+            {
+                "code": "1",
+                "label": "Missed at least one dose "
+            },
+            {
+                "code": "2",
+                "label": "Missed all doses"
+            }
+        ],
+        "field_note": "",
+        "text_validation_type_or_show_slider_number": "",
+        "text_validation_min": "",
+        "text_validation_max": "",
+        "identifier": "",
+        "branching_logic": "",
+        "required_field": "y",
+        "custom_alignment": "",
+        "question_number": "",
+        "matrix_group_name": "",
+        "matrix_ranking": "",
+        "field_annotation": "",
+        "evaluated_logic": ""
+    },
+    {
+        "field_name": "alcoholuse",
+        "form_name": "evening_questionnaire",
+        "section_header": "Alcohol Use",
+        "field_type": "radio",
+        "field_label": "How much alcohol have you consumed in the last 24 hours? ",
+        "select_choices_or_calculations": [
+            {
+                "code": "1",
+                "label": "None "
+            },
+            {
+                "code": "2",
+                "label": "Single small shot of spirits  (25ml, ABV 40%) 1 unit "
+            },
+            {
+                "code": "3",
+                "label": "Small glass of wine (125ml, ABV 12%)             1.5 units "
+            },
+            {
+                "code": "4",
+                "label": "Bottle of lager/beer/cider (330ml, ABV 5%)             1.7 units "
+            },
+            {
+                "code": "5",
+                "label": "Can of lager/beer/cider (440ml, ABV 5.5%)              2 units "
+            },
+            {
+                "code": "6",
+                "label": "Pint of lower-strength lager/beer/cider (ABV 3.6%)  2 units "
+            },
+            {
+                "code": "7",
+                "label": "Standard glass of wine (175ml, ABV 12%)              2.1 units "
+            },
+            {
+                "code": "8",
+                "label": "Pint of higher-strength lager/beer/cider (ABV 5.2%)   3 units "
+            },
+            {
+                "code": "9",
+                "label": "Large glass of wine (250ml, ABV 12%)               3 units"
```

Comment:
Can there be another way of spacing the label than the currently long whitspace?